.golangci.yml

```diff
@@ -1,27 +1,24 @@
 linters:
   enable:
-    - deadcode
     - errcheck
     - errorlint
     - cyclop
     - errname
     - exhaustive
     - exportloopref
     - gocritic
-    - goimports
+    - gofmt
```

Comment:
goimports complains with the generated sources. Will see if we can revert this change in a later version.